### PR TITLE
Bump PHP version to 5.6 for 3.4.

### PR DESCRIPTION
**Pull request by ADmad**
1st step to bumping PHP version to 5.6

### CODE REVIEW

**Comment by dereuromark** [comment]
Refs https://github.com/cakephp/cakephp/issues/9297
:+1:

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "source": "https://github.com/cakephp/cakephp"
     },
     "require": {
-        "php": ">=5.5.9",
+        "php": ">=5.6.0",
```
Comment (markstory) [inline composer.json]: We might have to update the zip files we use on AppVeyor as well. I think @lorenzo knows how that is done.

File: .travis.yml [hunk 2]
```diff
@@ -2,7 +2,6 @@ language: php
 
 php:
   - 7.0
```
Comment (dereuromark) [inline .travis.yml]: Shouldnt we order them ASC?

File: composer.json (hunk 1 above)
Comment (antograssiot) [inline composer.json]: @markstory that's easy we don't use a custom zip file anymore but it's downloaded from php website. I can do it if you want

Comment (antograssiot) [inline composer.json]: @admad the dropbox link is mine for the drivers in appveyor. may I update you PR or we first merge this one and I'll do an other one

File: .travis.yml (hunk 2 above)
Comment (antograssiot) [inline .travis.yml]: @dereuromark I think the idea was to trigger the coverage report ASAP if I remember well

Comment (ADmad) [inline .travis.yml]: Yes 7.0 is at top on purpose to get coverage report builds running first.

File: composer.json (hunk 1 above)
Comment (ADmad) [inline composer.json]: @antograssiot You can just tack on your commits to this PR

Comment (antograssiot) [inline composer.json]: @ADmad appveyor should be happy now. I didn't remember but the drivers for 5.6 were already in the zip file

**Pull request merged**